robot_smach_states/tutorial/challenge_tutorial.md

```diff
@@ -0,0 +1,236 @@
+# Creating a TU/e RoboCup challenge
+
+This tutorial explains how to setup a new RoboCup challenge. It assumes that you have finished the [ROS tutorials](http://wiki.ros.org/ROS/Tutorials) as well as the [Smach tutorials](http://wiki.ros.org/smach/Tutorials). Furthermore, it assumes that the simulator as well as 'middleware' are running. (ToDo: add links.)
+
+## Setting up the boiler plate
+
+### Setting up the package
+
+* Browse to the `tue_robocup` folder on your machine (ToDo: either create it directly in workspace, create link or whole target meuk):
+```
+cd ~/ros/$ROS_DISTRO/repos/https_/github.com/tue-robotics/tue_robocup.git/
+```
+* Create the package:
+```
+catkin_create_pkg challenge_example robot_skills robot_smach_states
+```
+This creates a ROS package called `challenge_example`. It depends on the `robot_skills`
+and the `robot_smach_states` package. This can be seen when inspecting the `package.xml` that is located inside your newly created `challenge_example` folder.
+* Make and source your `~/.bashrc` to make sure the package is in your `ROS_PACKAGE_PATH`.:
+```
```

Comment:
Create symbolic link in workspace?

Reply:
See line 9, but indeed could be extended

Reply:
Rewrote tutorial in b22087a5bb02c65398bcb173ad49805ef501e8d8 so that the package is created directly in your catkin workspace . I don't want to tire new users with the TU/e-specific stuff (should be a different tutorial)